packages/client/src/links/httpSubscriptionLink.ts

```diff
@@ -87,12 +90,16 @@ export function unstable_httpSubscriptionLink<
                 eventSource,
               },
             });
-
-            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-            eventSource!.removeEventListener('open', onStarted);
```

Comment:
I deleted this line since there was already an eventListener for this setup right? Why is there multiple?

Reply:
This removes it after it has started

Reply:
Oh wow I really am dyslexic